<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quan_Ly_Mon</class>
 <widget class="QDialog" name="quan_Ly_Mon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ff0000;&quot;&gt;Quản lý môn học&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timKiem">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: #F0F0F0;
color: black;
font-size: 16px;</string>
   </property>
   <property name="placeholderText">
    <string>                           Tìm Kiếm</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="dsachdiem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>641</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maMH">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: #F0F0F0;
color: black;
font-size: 16px;
</string>
   </property>
   <property name="placeholderText">
    <string>                   MÃ MH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tenMH">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: #F0F0F0;
color: black;
font-size: 16px;
</string>
   </property>
   <property name="placeholderText">
    <string>                     TÊN MÔN HỌC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="them">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 10px;
background-color: rgb(255, 35, 35);
color: white;
font-size: 16px;
font-weight: bold;
box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);</string>
   </property>
   <property name="text">
    <string>THÊM</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoiMaLop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loiTenMH">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="tinChi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 20px;
    background-color: #F0F0F0;
    color: black;
    font-size: 16px;
    padding: 2px;  /* Thêm khoảng cách giữa viền và nội dung */
    qproperty-alignment: 'AlignCenter';  /* Căn giữa văn bản */
}

QSpinBox::up-button, QSpinBox::down-button {
    width: 0px;
    height: 0px;
    border: none;
}


</string>
   </property>
  </widget>
  <widget class="QLabel" name="loiTinChi">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Số tín chỉ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>quan_Ly_Mon</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>quan_Ly_Mon</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
